aireview/src/aireview/domain/services/code_analyzer.py

```diff
@@ -16,14 +17,14 @@ def analyze_code(self, pull_request: PullRequest) -> Review:
         # self._ai_agent.initialize(f"Creating context for review of pull request {pull_request.number}")
         for index, file in enumerate(pull_request.files):
             prompt = self._build_analysis_prompt(file)
-            print(f"prompt: {prompt}")
             analysis = self._ai_agent.analyse(prompt)
-            print(f"analysis: {analysis}")
-            review_comments.append(self._parse_analysis(analysis))
-        return Review(1, review_comments,"summary of review", "COMMENT")
+            comments = self._parse_analysis(analysis)
+            review_comments.extend(comments)
+        return Review(pull_request.number, review_comments,"summary of review", "COMMENT")
```

Comment:
**style**: Using pull_request.number instead of hardcoded value 1 improves code reliability and maintainability.